packages/loot-core/src/client/shared-listeners.ts

```diff
@@ -95,21 +103,22 @@ export function listenForSyncEvent(actions, store) {
           // Tell the user something is wrong with the key state on
           // the server and the key needs to be recreated
           notif = {
-            title: 'Actual has updated the syncing format',
-            message:
+            title: i18n.t('Actual has updated the syncing format', {}),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
            title: i18n.t('Actual has updated the syncing format'),
```